AUTHORS

```diff
@@ -7,11 +8,13 @@ Kitson Kelly <dojo@kitsonkelly.com>
 Hsiaoming Yang <me@lepture.com>
 MinRK <benjaminrk@gmail.com>
 Isaac Z. Schlueter <i@izs.me>
+Ben Briggs <beneb.info@gmail.com>
 Ian Storm Taylor <ian@ianstormtaylor.com>
 Brad Harris <bmharris@gmail.com>
 Maksim Kozhukh <mkozhukh@ya.ru>
 Martin Zagora <zaggino@gmail.com>
 Matias Niemelä <matias@yearofmoo.com>
+Eugene Sharygin <eush77@gmail.com>
```

Comment:
Yeah, you can ignore those, I can either change this PR or you can remove them before squashing  :+1: